plasmapy/formulary/tests/test_collisions.py

```diff
@@ -1698,3 +1700,347 @@ def test_kwarg_method_error(self):
+                {
+                    "T_a": 1 * u.eV,
+                    "T_b": 1e2 * u.eV,
+                    "n_b": 1e10 * u.cm**-3,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+            ),
+            (
+                "i|i",
+                "slow",
+                (Particle("Na+"), Particle("Cl-")),
+                {
+                    "T_a": 1e2 * u.eV,
+                    "T_b": 1e4 * u.eV,
+                    "n_b": 1e20 * u.cm**-3,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+            ),
+            # Fast limit (x >> 1)
+            (
+                "e|e",
+                "fast",
+                (Particle("e-"), Particle("e-")),
+                {
+                    "T_a": 1e2 * u.eV,
+                    "T_b": 1e-1 * u.eV,
+                    "n_b": 1e20 * u.cm**-3,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+            ),
+            (
+                "e|i",
+                "fast",
+                (Particle("e-"), Particle("Na+")),
+                {
+                    "T_a": 1e-4 * u.eV,
+                    "T_b": 1e-3 * u.eV,
+                    "n_b": 1e20 * u.cm**-3,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+            ),
+            (
+                "i|e",
+                "fast",
+                (Particle("Na+"), Particle("e-")),
+                {
+                    "T_a": 1e4 * u.eV,
+                    "T_b": 1e-3 * u.eV,
+                    "n_b": 1e20 * u.cm**-3,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+            ),
+            (
+                "i|i",
+                "fast",
+                (Particle("Na+"), Particle("Cl-")),
+                {
+                    "T_a": 1e4 * u.eV,
+                    "T_b": 1e2 * u.eV,
+                    "n_b": 1e20 * u.cm**-3,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+            ),
+        ],
+    )
+    def test_limit_values(
+        self,
+        interaction_type,
+        limit_type,
+        constructor_arguments,
+        constructor_keyword_arguments,
+    ):
+        """Test the return values"""
+
+        value_test_case = CollisionFrequencies(
+            *constructor_arguments, **constructor_keyword_arguments
+        )
+
+        coulomb_density_constant = (
+            constructor_keyword_arguments["Coulomb_log"].value
+            * constructor_keyword_arguments["n_b"].value
+        )
+
+        expected_limit_values = self.get_limit_value(
+            interaction_type, limit_type, value_test_case
+        )
+
+        for i, (attribute_name, expected_limit_value) in enumerate(
+            zip(self.return_values_to_test, expected_limit_values)
+        ):
+            calculated_limit_value = getattr(value_test_case, attribute_name).value
+
```

Comment:
Explain why this is being done - otherwise it looks a little odd why this one is in different units than the others.

```suggestion
# Energy loss limit value is already in units of frequencies because of the way it is calculated
```